Sources/Cocoa/Components/LabelTextView.swift

```diff
@@ -141,3 +142,32 @@ extension LabelTextView {
         set { super.linkTextAttributes = newValue }
     }
 }
+
+// MARK: - Accessibility
+
+extension LabelTextView {
+    /// The accessibility trait value that the element takes based on the
+    /// text style it was given when created.
+     open override var accessibilityTraits: UIAccessibilityTraits {
+        get {
+            guard
+                let currentFont = font,
+                let textStyle = currentFont.fontDescriptor.object(forKey: .textStyle) as? UIFont.TextStyle
```

Comment:
This logic is being repeated in multiple places. Could you please create an extension on `UIFont` and determine if given font has title type and return a boolean. Then, use that boolean instead here.

```swift
font.isTextStyleHeader ? .header : super.accessibilityTraits
```